build.py

```diff
@@ -13,5 +13,9 @@
     os.system("conan package .. --build_folder=../")
     print(os.listdir("."))
     print(os.listdir("./lib"))
+    os.system("cd .. && conan export user/channel") # This will improve next 0.25 with an --cwd=.. arg
+    os.system("conan package_files Test/0.1@user/channel -f")  # -f in case it exists, overwrite
+    os.system("conan search Test/0.1@user/channel") # To show the package binary
+    # os.system("conan upload Test/0.1@user/channel --all -r=")
```

Comment:
Ok, that makes sense. Then it could be the easier for your project if your current build tooling is able to find the headers and libraries in such user folder. You might be interested in the new ``folder`` argument to imports, so all imports are organized in folders with the name of the package, so you avoid possible collisions of files with the same name.

In conan, as you might know, the approach is basically the opposite, instead of having all consumers of a library figuring out and configuring how to link with dependencies, it is the package creator that defines that via the ``package_info()`` method: the library names, the include paths, some preprocessor definitions that the consumers might need to use... This info is later translated to a file via generators, and the consumer build system use this file to load those values. If you would like to implement this approach, you can create and use custom generators for your build system, please ask for more details if interested.